pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -1450,40 +1270,23 @@ func (fs *eosfs) createNominalHome(ctx context.Context) error {
 		return errors.Wrap(err, "eosfs: error verifying if user home directory exists")
 	}
 
-	err = fs.createUserDir(ctx, u, home, false)
-	if err != nil {
-		err := errors.Wrap(err, "eosfs: error creating user dir")
-		return err
-	}
-
 	// set quota for user, depending on its type
 	quotaBytes := fs.conf.DefaultQuotaBytes
```

Comment:
If we no longer handle the home creation here, maybe it would make more sense to pass the type of account to the script, and it's there that we decide the quota we give and not here?

Reply:
Then we don't need to update reva to change the quota logic.

Reply:
I had exactly the same thought, but yet preferred to have the "policy brain" in Reva so the script is a simple "agent".

Thinking at this again now, and with the idea that possibly the lifecycle action is called outside of Reva, it actually makes sense to externalize the policy as well, otherwise we may have duplication of policy configuration. The script will include and "own" the quota policy then.

Reply:
I just realised that we might not even want to pass the type of account and let the account be resolved in the script

Reply:
Agreed to keep it, as it's harder to get the information in the script.